src/electron/main.ts

```diff
@@ -22,13 +38,69 @@ function createWindow(): void {
 
 	// and load the index.html of the app.
 	win.loadURL(
-		url.format({
-			pathname: PathUtils.join(__dirname, 'renderer', 'app.html'),
+		Url.format({
+			pathname: Path.join(__dirname, 'app.html'),
 			protocol: 'file:',
 			slashes: true
 		})
 	);
 
+	// Cast getPort return type from PromiseLike<number> to Promise<number>
+	// to avoid async-promise tslint rule to produce errors here
+	const port = await (getPort({ port: 1879 }) as Promise<number>);
+	const server = await createServer({ port });
+
+	// tslint:disable-next-line:no-any
+	const send = (message: any) => {
```

Comment:
We could type this a bit:
`const send = (message: { id: string; payload: any; type: string }) => {...`
Maybe also put these typings in there own interface